html/captive-portal/lib/captiveportal/PacketFence/Model/Portal/Session.pm

```diff
@@ -104,9 +106,10 @@ sub ACCEPT_CONTEXT {
     my $mgmt_ip = $management_network->{'Tvip'} || $management_network->{'Tip'} if $management_network;
     $destination_url = $request->param('destination_url') if defined($request->param('destination_url'));
 
-    if( $r->isa('Apache2::Request') &&  defined ( my $last_uri = $r->pnotes('last_uri') )) {
+    if( defined ( my $last_uri = $r->pnotes('last_uri') )) {
```

Comment:
Ok that only causes a problem in the catalyst server script so I will look into this later.
Since it does not cause a problem with the apache based server.